client/client-multi/client-c/src/kaa_event.h

```diff
@@ -33,17 +33,28 @@ extern "C" {
 #include <stddef.h>
 #include <stdint.h>
 #include "kaa_error.h"
-
-#define KAA_ENDPOINT_ID_LENGTH 20
-
-typedef uint8_t        kaa_endpoint_id[KAA_ENDPOINT_ID_LENGTH];
-typedef const uint8_t* kaa_endpoint_id_p;
+#include "platform/ext_event_listeners.h"
 
 typedef void (*kaa_event_callback_t)(const char *event_fqn, const char *event_data, size_t event_data_size, kaa_endpoint_id_p event_source);
 typedef size_t kaa_event_block_id;
 
 typedef struct kaa_event_manager_t kaa_event_manager_t;
 
+
+/**
+ * @brief Initiates search of available event listeners by given FQNs.
```

Comment:
"Initiates" gives an impression that the procedure must be somehow stopped. Please clarify.